WikiTemplate/Crypto/wikitemplate-cryptoiOS.md

```diff
@@ -0,0 +1,60 @@
+
+## Onboarding
+
+- [ ] Verify able to create a wallet by following onboarding
+- [ ] Verify not able to screenshots during onboarding screens
+- [ ] Verify able to backup wallet seed phrase during onboarding
+- [ ] Verify able to skip backup wallet seed screen to complete onboarding
+- [ ] Verify onboarding is shown when trying to interact with Dapp without creating wallet
+
+## Panel
+
+- [ ] Verify panel is only shown in URL when a Dapp is trying to interact
+- [ ] Verify able to change default network from panel
+- [ ] Verify able to change accounts from panel
+- [ ] Verify notification dot is shown when panel when an action is pending for Wallet
+
+## Ethereum
+
+- [ ] Verify Ethereum account is created as soon as onboarding is completed
+- [ ] Verify able to create Ethereum account can be created from accounts tab
+- [ ] Verify able to import Ethereum account via Private Key/JSON file
+- [ ] Verify able to submit an ETH transaction
+
+## Solana
+
+- [ ] Verify able to create a Solana account when Solana Mainnet Beta is selected from network picker
+- [ ] Verify able to create a Solana account from accounts tab
+- [ ] Verify able to send tokens from one Solana account to another
+- [ ] Verify able to submit a SOL transaction
+
+## Network Settings
+
+- [ ] Verify able to add a custom network via Settings
+- [ ] Verify able to change active network to a custom network via portfolio
+
+## Custom Tokens
+
+- [ ] Verify able to manually add a custom token
+- [ ] Verify able to manually add ERC721 tokens
+- [ ] Verify able to add custom tokens via webpage (Coingecko/CoinMarketCap)
+
+## Dapp Support (Ethereum)
+
+- [ ] Verify able to connect to a Dapp
+- [ ] Verify when a Dapp is trying to interact a toast notification is shown
+- [ ] Verify clicking on the Dapp interaction notification, opens the Panel for Dapp interaction
+- [ ] Verify able to connect multiple accounts to a Dapp  
+- [ ] Verify able to connect/disconnect to Dapp from Panel
```

Comment:
probably want to add some verification for dapps requests?
1. connection
2. pending transaction
3. add chain
4. switch chain
5. add suggested token
6. sign message
7. get encryption public key
8. decrypt

Reply:
Added the new tests